weave/index/index_pandas.py

```diff
@@ -12,6 +12,27 @@
 from .create_index import create_index_from_fs
 from .index_abc import IndexABC
 
+def slice_df(df, max_rows=None, offset=0):
```

Comment:
In this function I think it would be a good idea to raise some type of warning or at least print statement for when the number of rows in the index exceeds the max_rows that the user specifies. You could state the number of rows that were cut off from your df. This way the user knows how many rows are being cut off from the index.

Reply:
I think this is a really good suggestion. Are you ok if we add this as a user suggestion?